### PR TITLE
updating footers across sites to include AWS, Github, and Google Clas…

### PR DESCRIPTION
**Pull request by annaxuphoto**
Footer updates according to the [spec](https://docs.google.com/document/d/1f-hFcTOBSJr5rtAxasmwlnAFc3LWfXjwYygo9nIfMVs/edit#) to include AWS, Github, and Google Classroom copyrights and trademarks.

Dashboard:
Screen Shot 2020-11-17 at 5 38 19 PM

Screen Shot 2020-11-18 at 1 14 12 PM

Pegasus:
Screen Shot 2020-11-17 at 5 42 35 PM

pegasus medium footer
pegasus mobile footer

Hourofcode.com:
Screen Shot 2020-11-17 at 5 54 35 PM

Screen Shot 2020-11-17 at 7 56 31 PM

I added the AWS and Github strings to the [Pegasus I18n sheet](https://docs.google.com/spreadsheets/u/1/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0) as well.

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dju90** [comment]
I like the amount of spacing above the AWS mark in the grey dashboard small footer and the 2nd & 3rd pegasus screenshots. Would it be possible to do the same for the other views represented here?

**Comment by dju90** [comment]
One small change: in the dashboard 1st screenshot, let's put the AWS copy inline with the rest of the copyright info (so right after the Google Inc. copy)

**Comment by annaxuphoto** [comment]
Done! I've updated the screenshots accordingly.

**Comment by dju90** [comment]
Hmm, when I pulled the branch, I'm getting a "translation missing" error on localhost.code.org:3000
image

**Comment by annaxuphoto** [comment]
That's because the changes in the [G sheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0) haven't been scooped yet! I made the string changes locally in https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/cache/i18n/en-US.yml

**Comment by dju90** [comment]
Nit: the spacing below the mark on the last hourofcode.com screenshot (mobile) is a little sparse, and a pretty big gap above it. Any chance we can make it a bit more balanced?

**Comment by annaxuphoto** [comment]
Updated the commit and screenshots with different margins

File: dashboard/app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -485,6 +485,10 @@ html, body {
   a:hover {
     text-decoration: underline;
   }
+  .powered-by-aws {
+    width: 100%;
+    margin-top: 2%;
```
Comment (daynew) [inline dashboard/app/assets/stylesheets/application.scss]: *nit* I don't think we use `%` much. I have mostly seen `px`. Why did you choose % over px?

**Review by daynew — APPROVED** [review]
Looks good code wise!
Double check with Bryan that this look is okay for the copyright:
image
I got this screenshot by:
1. Go to http://localhost-localhost-studio.code.org:3000/s/dance-2019/stage/1/puzzle/10
1. Click `Run`
1. Click `Finish`
1. Copy/Paste the Share URL into your browser (this mimics when students share their project with family/friends)
1. Click `Build on Code Studio` in the bottom right
1. Click `Copyright`

**Comment by dju90** [comment]
Last adjustment: looking at the **dashboard small** footers, it actually makes more sense for the Google Classroom and Amazon copyright info to be in the same text block as the other copyright reservations (on a new line, but without any whitespace between lines). Similar in intent to how the full-size dashboard footer treats the legalese. So it would go:
.
.
.
Plants vs Zombies...
Gumball...
Google Classroom...
Amazon Web Services...  
[vertical space]
p5.play
[vertical space]
Built on GitHub from Microsoft
[vertical space]
AWS image

Sorry I didn't catch this earlier.

File: dashboard/app/assets/stylesheets/application.scss (hunk 1 above)
Comment (annaxuphoto) [reply dashboard/app/assets/stylesheets/application.scss]: I can change this to px! In the past I've liked to use relative spacing vs absolute, but no strong preferences

**Comment by annaxuphoto** [comment]
Updated screenshots & commits with altered whitespace and px instead of % in stylesheets

**Review by Hamms — COMMENTED** [review]
Is it intentional that this PR includes a change to the `schema_cache.dump`?

**Pull request merged**